cmd/flags.go

```diff
@@ -196,6 +196,16 @@ func clientFlags() []cli.Flag {
 			Value: 0.0,
 			Usage: "open files cache timeout in seconds (0 means disable this feature)",
```

Comment:
The period to reuse open file without checking update.

Reply:
Changed